ui/src/routes/incarnations/List.tsx

```diff
@@ -120,9 +126,13 @@ export const IncarnationsList = () => {
           <tr key={x.id}>
             <td>{x.id}</td>
             <td>
-              <CellText>{x.incarnationRepository}</CellText>
+              <Tooltip title={x.incarnationRepository}>
+                <CellText>{x.incarnationRepository}</CellText>
+              </Tooltip>
+            </td>
+            <td>
+              <CellText>{x.targetDirectory}</CellText>
```

Comment:
I'm just pattern matching this code, but what about tooltips for the target directory, too?

```suggestion
              <Tooltip title={x.targetDirectory}>
                <CellText>{x.targetDirectory}</CellText>
              </Tooltip>
```